explorer/explorer_windows.go

```diff
@@ -118,8 +120,61 @@ func (e *Explorer) importFile(extensions ...string) (io.ReadCloser, error) {
 	return os.Open(path)
 }
 
-func (e *Explorer) importFiles(_ ...string) ([]io.ReadCloser, error) {
-	return nil, ErrNotAvailable
+func (e *Explorer) importFiles(extensions ...string) ([]io.ReadCloser, error) {
+	pathUTF16 := make([]uint16, _FilePathLength)
+
+	open := _OpenFileName{
+		File:       &pathUTF16[0],
+		MaxFile:    _FilePathLength,
+		Filter:     buildFilter(extensions),
+		Flags:      _FlagFileMustExist | _FlagForceShowHidden | _FlagDisableLinks | _FlagAllowMultiSelect | _FlagExplorer,
+		StructSize: _OpenFileStructLength,
+	}
+
+	if r, _, _ := _GetOpenFileName.Call(uintptr(unsafe.Pointer(&open))); r == 0 {
+		return nil, ErrUserDecline
+	}
+
+	// Split the pathUTF16 by null characters
+	paths := make([]string, 0)
+	currentPath := make([]uint16, 0)
+	for _, char := range pathUTF16 {
+		if char == 0 {
+			if len(currentPath) > 0 {
+				paths = append(paths, windows.UTF16ToString(currentPath))
+				currentPath = make([]uint16, 0)
```

Comment:
I think it's better to use `currentPath = currentPath[:0]` so you re-use the same buffer and (in the best case) avoids an unnecessary allocation.